File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    printf("How many bricks tall does the mario's pyramid will have?\n");
    int height = get_int("(insert a number between 1 and 8): ");
    
    if (height < 1 || height > 8)
    {
        do
        {
            height = get_int("(insert a number between 1 and 8): ");
        }
        while (height < 1 || height > 8);
    }

    for (int i = 1; i <= height; i++) //lines
    {
        //first pyramid
        for (int j = 1; j <= height; j++)
        {
            if (j + i > height) //condition to brick
            {
                printf("#"); //brick
            }
            else
            {
                printf(" "); //empty
            }
        }
        
        //gap
        printf("  ");

        //second pyramid
        for (int j = height + 3; j <= (2 * height) + 2; j++)
        {
            if (j <= (i + height + 2))
            {
                printf("#"); //brick
            }
            
        }
        printf("\n"); //new line
    }
}

File: mario.py
# função para ler input de inteiro do usuário
from cs50 import get_int

# ler inteiro do user
height = get_int("Height: ")

# checking if height is between 1 and 8
while height < 1 or height > 8:
    height = get_int("Height: ")

# building the pyramid
for i in range(height):

    # first half
    print(" " * (height - 1 - i), end="")
    print("#" * (i + 1), end="")

    # gap
    print("  ", end="")

    # second half
    print("#" * (i + 1))
